packages/providers/test/PoapMomentsApi.spec.ts

```diff
@@ -132,5 +132,23 @@ describe('PoapMomentsApi', () => {
       api = new PoapMomentsApi({});
       await expect(api.createMoment(createMomentInput)).rejects.toThrow();
     });
+
+    it('should allow Moment creation without any drop', async () => {
+      const mockResponse: CreateMomentResponse = {
+        id: '1',
+        author: '0x1234',
+        createdOn: '2023-01-01T00:00:00.000Z',
+        dropIds: [],
```

Comment:
This test could also test when is `undefined` --the new expected value.

Reply:
This is the response. The input has no `dropIds` property.